src/services/ProductsAPI.js

```diff
@@ -0,0 +1,8 @@
+import API from "@/services/API";
+export default {
+    postProduct: (product) => API().post("/api/product", { product }),
+    getProduct: () => API().get("/api/product"),
+    deleteProduct(id) {
+        return API().delete("/api/product/" + id);
+    },
```

Comment:
hier gebruik je best de arrow notation
```suggestion
    deleteProduct: id => API().delete("/api/product/" + id),
```